wsit/ws-sx/wssx-impl/src/main/java/com/sun/xml/wss/impl/transform/AttachmentContentOnlyTransform.java

```diff
@@ -61,4 +69,9 @@ protected XMLSignatureInput enginePerformTransform(
    public boolean wantsNodeSet ()       { return true; }
    public boolean returnsOctetStream () { return true; }
    public boolean returnsNodeSet ()     { return false; }
+
+    // Required by Santuario 2.2.X
+    protected XMLSignatureInput enginePerformTransform(XMLSignatureInput input, OutputStream os, Element transformElement, String baseURI, boolean secureValidation) throws IOException, CanonicalizationException, InvalidCanonicalizerException, TransformationException, ParserConfigurationException, SAXException {
+        return null;
```

Comment:
it is always needed to nullify the result?

Reply:
See upstream

Reply:
This method is only required in Santuario 2.2.X which upstream doesn't use